Game/Game/classes/Notepad.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Game.extensions;
+
+namespace Game.classes
+{
+    public class Notepad : Player
+    {
+        private int[] _array;
+        public Notepad(string name) : base(name)
+        {
+            _array = new int[MAX_VALUE - MIN_VALUE];
+            for (int i = MIN_VALUE; i < MAX_VALUE; i++)
+            {
```

Comment:
code duplication with Cheater
try to move it to the separate class and inject (:

Reply:
Fixed